repository/c.json

```diff
@@ -4852,6 +4852,17 @@
 				}
 			]
 		},
+		{
+			"name": "Cyclone SyntaxHighlighting",
```

Comment:
Language packages should just be named after the language, it does a lot more than just highlighting.